content/en/account_management/authen_mapping/_index.md

```diff
@@ -0,0 +1,498 @@
+* **`role_uuid`** [*required*, *default*=none]:
+  The Roles API can be used to create and manage Datadog roles, what global permissions they grant, and which users belong to them. When you create a Role, it is assigned a UUID. For more information about finding the `role_uuid` for the role you are updating, see the [Role API documentation][1].
+* **`attribute_key`** [*required*, *default*=none]:
+ The `attribute_key` and the `attribute_value` are key/value pairs defined in their SAML assertions from their Identity Providers. You can define these for your own use case. For example, `attribute_key` could be `member-of` and the `attribute_value` could be `Development`.
+* **`attribute_value`** [*required*, *default*=none]:
+  The `attribute_key` and the `attribute_value` are key/value pairs defined in their SAML assertions from their Identity Providers. You can define these for your own use case. For example, `attribute_key` could be `member-of` and the `attribute_value` could be `Development`.
+
+{{< tabs >}}
+{{% tab "Example" %}}
+
+```sh
+curl -X POST \
+         "https://app.datadoghq.com/api/v2/authn_mappings/{UUID}" \
+         -H "Content-Type: application/json" \
+         -H "DD-API-KEY: <YOUR_DATADOG_API_KEY>" \
+         -H "DD-APPLICATION-KEY: <YOUR_DATADOG_APPLICATION_KEY>" \
+         -d '{
+             "data": {
+                 "type": "authn_mappings",
+                 "id": "123e4567-e89b-12d3-a456-426655440000",
+                 "attributes": {
+                      "role_uuid": "123e4567-e89b-12d3-a456-426655445555",
+                      "attribute_key": "string",
+                      "attribute_value": "string"
+                }
+             }
+         }'
+```
+{{< /code-block >}}
+
+Replace the `<YOUR_DATADOG_API_KEY>` and `<YOUR_DATADOG_APPLICATION_KEY>` placeholder with the corresponding [API and application keys for your organization][1].
+
+[1]: https://app.datadoghq.com/account/settings#api
+{{% /tab %}}
+{{% tab "Response" %}}
+
+{{< code-block lang="json" filename="response.json" disable_copy="true" >}}
+{
+  "data": {
+    "attributes": {
+      "created_at": "timestamp",
+      "created_by": "string",
+      "role_uuid": "00000000-0000-0000-0000-000000000000",
+      "saml_assertion_attribute_id": 0
+    },
+    "type": "authn_mappings",
+    "id": "string",
+    "relationships": {
+      "saml_assertion_attributes": {
+        "data": {
+          "id": 0,
+          "type": "saml_assertion_attributes"
+        }
+      },
+      "roles": {
+        "data": {
+          "id": "123e4567-e89b-12d3-a456-426655440000",
+          "type": "roles"
+        }
+      }
+    }
+  },
+  "included": [
+    {
+      "data": {
+        "id": "string",
+        "type": "roles",
+        "attributes": {
+          "created_at": "string",
+          "modified_at": "string",
+          "uuid": "123e4567-e89b-12d3-a456-426655440000",
```

Comment:
Not your fault but changes are pending to remove the uuid field from roles resources, meaning we would have to delete `uuid` values from any type role response bodies here. These hasn't gone live quite yet, but since we want users to use the `id` field instead (since those are always equal to one another), I think we might want to remove this here preemptively